src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./src/styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__cell calendar__cell--1"></div>
+      <div class="calendar__cell calendar__cell--2"></div>
+      <div class="calendar__cell calendar__cell--3"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
```
There is actually no need to hardcode modifier for each cell. You can use the index inside loop for :nth-child() pseudoclass